File: practica n6/estudiante.py
class Estudiante:
    def __init__(self, codigo: str, nombre: str):
        self.codigo = codigo
        self.nombre = nombre


    def mostrar_info(self) -> None:
        print(f"Estudiante: {self.nombre}, Codigo: {self.codigo}")

    def __str__(self) -> str:
        return f"{self.nombre} [{self.codigo}]"

File: practicas/bibli/estudiante.java
package practicas.bibli;

public class estudiante {
    private String codigo;
    private String nombre;

    public estudiante(String codigo, String nombre) {
        this.codigo = codigo;
        this.nombre = nombre;
    }

    public void mostrarInfo() {
        System.out.println("Estudiante: " + nombre + ", Codigo: " + codigo);
    }
}
